src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -363,7 +363,6 @@ private function parseDefinition($id, $service, $file, array $defaults)
             if (null !== $inheritTag) {
                 $definition->setInheritTags($inheritTag);
             }
-            $defaults = array();
```

Comment:
This is below the condition : `elseif (isset($service['parent'])) {`.

Emptying `$defaults` meant that services with a parent did not have `_defaults` applied. I think the intention was to not double-apply defaults (since the parent will already have them applied). Removing this causes double tags (as I described in my description). BUT, I think this was a bug. What if I had a child in this file, but my parent in another file? In that case, the neither the child nor the parent would have the `_defaults` applied.